com/edocti/jintro/lab02/Merger.java

```diff
@@ -0,0 +1,58 @@
+package com.edocti.jintro.lab02;
+public class Merger {
+	
+    private static boolean isSorted(int[] a) {
+        for (int i = 1; i <a.length; 1++) {
+            if (a[i] < a[i-1]) { // preconditii
+                return false; // preconditii
+            }
+        }
```

Comment:
The function should `return` on all possible branches => you need to `return` something in case the array is sorted